noir-projects/aztec-nr/aztec/src/macros/aztec.nr

```diff
@@ -268,9 +268,9 @@ comptime fn generate_process_log() -> Quoted {
     if notes.len() > 0 {
         quote {
             unconstrained fn process_log(
-                log_ciphertext: BoundedVec<Field, aztec::encrypted_logs::log_encryption::PRIVATE_LOG_CIPHERTEXT_LEN>,
+                log_ciphertext: BoundedVec<Field, aztec::protocol_types::constants::PRIVATE_LOG_SIZE_IN_FIELDS>,
```

Comment:
As mentioned before this changed is caused by me re-introducing the tag here because I needed that info to figure out which logs are duplicates in PrivateEventDataProvider.